src/Entity/AppInfo.php

```diff
@@ -0,0 +1,95 @@
+<?php
+
+namespace App\Entity;
+
+use App\Repository\AppInfoRepository;
+use Doctrine\ORM\Mapping as ORM;
+
+#[ORM\Entity(repositoryClass: AppInfoRepository::class)]
+class AppInfo
+{
+    #[ORM\Id]
+    #[ORM\GeneratedValue]
+    #[ORM\Column]
+    private ?int $id = null;
+
+    #[ORM\Column(length: 255)]
```

Comment:
Shouldn't those columns be nullable? If so, then you would need to express it specifically, as the default of "nullable" = false , Isn't it?

Reply:
The migrations of those columns are NOT NULL by default, but ok I will make it looks more properly :)

Reply:
Ah, ok...nevermind my comment then, not so important then, my mistake